### DISCUSSION TITLE
Could PR AUC  be supported?

### DISCUSSION THREAD

**Discussion by ShixiangWang** [keras-team/autokeras#1622] [Q&A]
PR-AUC seems better than ROC-AUC for imbalance data with only a few positive cases, could you support this objective? I don't know how to implement this.

Best

**Comment by haifeng-jin** [comment]
We support custom metrics. Just try to create this custom metric using Keras.
https://autokeras.com/tutorial/faq/#how-to-use-customized-metrics-to-select-the-best-model
